macos/ONLYOFFICE/update/updates/ONLYOFFICE/changes/8.1.0/changes.ru.html

```diff
@@ -81,56 +81,90 @@
+            <li>Added a title and a button to close the panel to the <code>Chat</code> panel</li>
         </ul>
-        <h4>Редактор документов</h4>
+        <h4>Document Editor</h4>
         <ul>
-            <li>TODO</li>
+            <li>New button on the top toolbar for changing document editing mode: <code>Editing</code>, <code>Reviewing</code>, <code>Viewing</code></li>
+            <li>Added tooltips for new or updated functionality (displayed when loading the editor or when switching to the corresponding tab)</li>
+            <li>Implemented the ability to set the format for page numbering</li>
+            <li>Added support for the page color</li>
+            <li>Updated built-in paragraph styles</li>
+            <li>New items in the indents menu for opening the right panel and managing paragraph indents, the ability to manage paragraph indents via the top toolbar</li>
+            <li>Color theme button is now on the Layout tab</li>
+            <li>Mail merge button is now on the Collaboration tab</li>
+            <li>Line spacing options updated</li>
+            <li>Improved work of the algorithm for displaying numbers and punctuation in Arabic text for the Neutral and Weak classes</li>
+            <li>Improved fitting for paragraphs with main <code>RTL</code> direction</li>
         </ul>
-        <h4>Редактор таблиц</h4>
+        <h4>Spreadsheet Editor</h4>
         <ul>
-            <li>TODO</li>
+            <li>New languages added: ligature support</li>
+            <li>Selected cells are highlighted on their respective row/column numbers</li>
+            <li>New functions: <code>GETPIVOTDATA</code>, <code>IMPORTRANGE</code></li>
+            <li>New function category: Custom based on jsdoc</li>
+            <li>Version history update: edited cells are highlighted</li>
+            <li>Users get custom protected range cells viewing rights</li>
+            <li>Implemented the ability to copy/move sheets between books in one browser</li>
+            <li>Changed the appearance of the sheet list in the embedded viewer in accordance with the styles of the main spreadsheet editor</li>
         </ul>
-        <h4>Редактор презентаций</h4>
+        <h4>Presentation Editor</h4>
         <ul>
-            <li>TODO</li>
+            <li>New slide settings on the right panel: show background graphics, reset background to the theme background, apply settings to all slides</li>
+            <li>Added <code>Animation</code> pane</li>
+            <li>Added a mode for editing master slides and templates</li>
         </ul>
-        <h4>Формы</h4>
+        <h4>Forms</h4>
         <ul>
-            <li>TODO</li>
+            <li>When adding a fixed form, now it is inserted without wrapping in front of the text</li>
+            <li>The color of the Picture placeholder corresponds to the color of the role for this form</li>
+            <li>The thickness of the frame for required fields is now 2px with any zoom</li>
```

Comment:
<li>The thickness of the frame for required fields is now 2 px with any zoom</li>